recipe/meta.yaml

```
@@ -22,16 +22,18 @@ requirements:
     - setuptools-scm
     - wheel
   run:
-    - numpy
     - python
+    - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy
```
As it doesn't rely on `numpy` in `host` then we can use a plain `numpy` as it's defined in the requirements https://github.com/dfm/emcee/blob/v3.1.4/setup.py#L25. `pin_compatible` is used when you have numpy in host. That will get correct pinnings for different Python versions at runtime

---

I'll try without pinning